### ISSUE TITLE
Intel+Nvidia "Compute shaders not supported"

### ISSUE THREAD

**Issue by zzaziki** [tobspr/RenderPipeline#75]
Hi.

This might sound silly, but it got me very frustrated. I'm trying to setup.py but I get an error that compute shaders are not supported. I don't have a powerful video card but I do think they are supported. Anyway my laptop has a Nvidia 650M which should support compute.

My assumption is that the Intel "graphic card" is used when testing for compute shaders. Unfortunately, I cannot completely disable that card as the Nvidia card is a slave to the Intel card, if I disable Intel card Nvidia becomes unavailable too. No BIOS options to disable it either. All I can do externally from windows or Nvidia control panel is to 'prefer' the Nvidia card, but that goes right past panda3d.

I tried a lot to find a way to setup panda3d or this plugin or my computer to make the second graphic card the default graphic card, but I'm just not seeing it. *How do I do that?*

Thanks!

**Comment by rdb** [comment]
I have a laptop with NVIDIA 650M.  If you go to the NVIDIA control panel and tell it to globally prefer your NVIDIA card (or just enable it for python.exe) it should properly choose the NVIDIA card.

**Comment by zzaziki** [comment]
TL;DR: problem solved

Long version: It just boggles my mind that I tweaked and searched all those Nvidia control panel but could not see the proper setting. There is a full list of settings but actually just above that list is the globally preferred adapter. Just you saying that it should be there made me look extra careful, and surprise, it's there..

Thanks for the lightning fast response 👍
